packages/cdk/lib/stacks/engines/miniwdl-engine-construct.ts

```diff
@@ -62,6 +70,11 @@ export class MiniwdlEngineConstruct extends EngineConstruct {
               actions: ["tag:GetResources"],
               resources: ["*"],
             }),
+            new PolicyStatement({
+              effect: Effect.ALLOW,
+              actions: ["batch:TerminateJob"],
+              resources: ["*"],
```

Comment:
I believe that we tag all of our instances with the following key value pair
```
"application-name": "agc"
```

Can we restrict the policy to just those instances (we should do this for the other engines as well)?

Reply:
This statement is attached to the WES adapter policy. Currently we don't tag our head engine fargate tasks so we can't restrict them. However, I will raise a new issue to do this because if we did then we could reduce access with something like. 

```
conditions: { StringEquals: { "aws:ResourceTag/application-name": "agc" } }
```